src/main/java/org/folio/inventory/domain/AsynchronousCollection.java

```diff
@@ -30,4 +31,10 @@ void delete(String id,
   void update(T item,
               Consumer<Success<Void>> completionCallback,
               Consumer<Failure> failureCallback);
+
+  default void addBatch(List<T> items,
+                        Consumer<Success<BatchResult<T>>> resultCallback,
+                        Consumer<Failure> failureCallback) {
+    throw new UnsupportedOperationException();
```

Comment:
Why has a default implementation been added? I think this should be an active decision by each implementor of the interface. If some of the implementors aren't going to implement this, then maybe it should be a separate interface?

Reply:
thank you for review, we will take note of this. I created the [task](https://issues.folio.org/browse/MODINV-154) to make refactoring in this way.